bcipy/tasks/rsvp/icon_to_icon.py

```diff
@@ -18,6 +19,7 @@
 from bcipy.helpers.triggers import write_triggers_from_sequence_icon_to_icon
 from bcipy.tasks.task import Task
 
+log = logging.getLogger(__name__)
```

Comment:
The Task is constructed with a logger

Reply:
The logger was needed for the RandomLm object. However, now that it's factored out I can remove this statement.